.github/workflows/github-pages.yml

```diff
@@ -1,3 +1,17 @@
+# Copyright 2023 Google LLC
```

Comment:
Oops, thanks for fixing this.

Reply:
It was auto-fixed for me by build_and_test :)